File: my-submissions/e27.java
// https://leetcode.com/problems/remove-element/


class Solution {
    public int removeElement(int[] nums, int val) {
        int pointer = 0;

        for (int i = 0; i < nums.length; i++) {
            if (nums[i] != val) {
                nums[pointer] = nums[i];
                pointer += 1;
            }
        }
        return pointer;
    }
}

File: my-submissions/e27.py
# https://leetcode.com/problems/remove-element/


class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        currentPointer = 0
        for i in range(len(nums)) :
            if not nums[i] == val :
                nums[currentPointer] = nums[i]
                currentPointer += 1
        return currentPointer